### PR TITLE
adds token page navigation tests

### PR DESCRIPTION
**Pull request by warrenv**
### Description

- Sets up page object for tokens page and adds  tests to ensure that only logged in users can navigate to the tokens page.
- Refactors the dashboard page object to match structure used in other page objects.
- Stubs out a file for the token page tests.

### PR Review Checklist

* [ ] **Have you tested the changes?**
* [ ] **Is QE review required? If so, has this been reviewed by QE?** (For all new features, review by QE is required. Minor changes which already have passing functional tests may not need a new QE review -- when in doubt, loop in a QE reviewer).
* [ ] **Are there code changes that require new unit tests? If so, are those tests included?** (A separate PR for tests is not appropriate).
* [ ] **Are documentation updates required? If so, does this PR include it?** (Separate documentation PR is generally not appropriate).
* [ ] **Are the added/edited files named appropriately and located correctly in the project directory structure?**

### CODE REVIEW

File: tests/functional/features/navigation.js [hunk 1]
```diff
@@ -24,3 +25,27 @@ test(`On the home page, click to navigate to the login/register page`, async t =
...
+  const actual = await getLocation('pathname')
+
+  await t.expect(actual).eql(expected, should)
+})
+
+test(`A logged in user can access the api token page at ${pages.TOKEN}`, async t => {
+  const should = `Should load the token page at ${pages.TOKEN}`
+  const expected = pages.TOKEN
+  await t
+    .useRole(LoginRegisterPage.frostUser('fah@test.com', 'Foo@foo12345'))
```
Comment (jwicks31) [inline tests/functional/features/navigation.js]: This is showing that the "specified resource does not exist" error message when logging in for me.

Comment (warrenv) [reply tests/functional/features/navigation.js]: Ah, you are right. That's an account I created locally for testing. I'll need to figure out a way to setup a user account.

File: src/components/molecules/PanelOptions/PanelOptions.tsx [hunk 2]
```diff
@@ -14,6 +14,7 @@ export const PanelOptions = (props: PanelOptionsProps) => (
     <div className={'PanelOptions__box-buttons'}>
       <Link to={'/token'}>
         <BoxButton
+          className={'Panel CreateToken'}
```
Comment (wzalazar) [inline src/components/molecules/PanelOptions/PanelOptions.tsx]: the name of the classes don't follow the `BEM` convention

Comment (warrenv) [reply src/components/molecules/PanelOptions/PanelOptions.tsx]: And what name should be used be in this case?

Comment (warrenv) [reply src/components/molecules/PanelOptions/PanelOptions.tsx]: How come 'Panel' doesn't follow the BEM convention?

Comment (wzalazar) [reply src/components/molecules/PanelOptions/PanelOptions.tsx]: In my mind I think the block is `PanelOptions__box-buttons` and you have added a new element, so

B = block = PanelOptions__box-buttons
E =  element = `new_element`
M = modifier

What do you say? it's only a detail

**Pull request merged**